app/containers/wallet/WalletSummaryPage.js

```diff
@@ -76,15 +80,46 @@ export default class WalletSummaryPage extends Component<Props> {
       }
     }
 
+    const notification = this._getThisPageActiveNotification();
+
     return (
       <VerticalFlexContainer>
+
+        <NotificationMessage
+          icon={successIcon}
+          show={!!notification}
+        >
+          {!!notification && <FormattedHTMLMessage {...notification.message} />}
+        </NotificationMessage>
+
         <WalletSummary
           numberOfTransactions={totalAvailable}
           pendingAmount={unconfirmedAmount}
           isLoadingTransactions={recentTransactionsRequest.isExecutingFirstTime}
         />
+
         {walletTransactions}
+
       </VerticalFlexContainer>
     );
   }
+
+  _getThisPageActiveNotification = (): ?Notification => {
+    let notification = null;
+
+    const { mostRecentActiveNotification } = this.props.stores.uiNotifications;
+    if (mostRecentActiveNotification
```

Comment:
I would use `.include` and I think the if would be a little more clean

Reply:
nice idea